pkg/receive/hashring.go

```diff
@@ -296,6 +292,21 @@ func HashringFromConfig(algorithm HashringAlgorithm, replicationFactor uint64, c
 	if len(config) == 0 {
 		return nil, errors.Wrapf(err, "failed to load configuration")
 	}
+	
+	if (algorithm != "hashmod") && (algorithm != "ketama") {
+		log.Println("The specified algorithm is incorrect. Fall back to hashmod algorithm")
```

Comment:
Can we log more information about the hashring to help debugging & problem solving? For example, the hashring name and tenants could be useful.

Reply:
Sure!

Reply:
Thanks for your contribution, @haanhvu 🙇

Reply:
@douglascamata Do you think `log.Println` is good enough for all cases?

Reply:
Unfortunately, I don't think so, @haanhvu. It's better to use our chosen logging library (`"github.com/go-kit/log"`), which might require that you send down the logger as a parameter to this function and then do something like:

```
# import "github.com/go-kit/log/level"
level.Warning(logger).Log(...)
```

I think `Warning` is the best level as this doesn't stop Receive from starting up.